src/NHibernate/Transaction/AdoNetWithDistributedTransactionFactory.cs

```diff
@@ -1,227 +0,0 @@
-using System;
-using System.Collections;
-using System.Linq;
-using System.Transactions;
-using NHibernate.Engine;
-using NHibernate.Engine.Transaction;
-using NHibernate.Impl;
-
-namespace NHibernate.Transaction
-{
-	public class AdoNetWithDistributedTransactionFactory : ITransactionFactory
```

Comment:
Renamed. Unfortunately I have squashed that with too many changes, causing the history link with new file to be lost... I may re-split the commit to preserve history link.